features/raadpleeg-verblijfplaats-met-peildatum/gba/test/overlap-datum-aanvang-gba.feature

```diff
@@ -0,0 +1,108 @@
+#language: nl
+
+Functionaliteit: test dat raadplegen historie met periode bij een opeenvolgende verblijfplaatsen met gelijke of overlappende datum aanvang correct wordt geleverd
+
+    Achtergrond:
+      Gegeven adres 'A1' heeft de volgende gegevens
+      | gemeentecode (92.10) | straatnaam (11.10) |
+      | 0800                 | Korte straatnaam   |
+      En adres 'A2' heeft de volgende gegevens
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) | straatnaam (11.10) | naam openbare ruimte (11.15) | woonplaats (11.70) | identificatiecode verblijfplaats (11.80) | identificatiecode nummeraanduiding (11.90) |
+      | 0800                 | 0800010000000002                         | Korte straatnaam   | Officiele straatnaam         | Testdorp           | 0800010000000002                         | 0800200000000002                           |
+
+  Rule: een verblijfplaats wordt niet geleverd wanneer de bekende datum aanvang volgende verblijfplaats gelijk is aan (of eerder) dan de datum aanvang van het verblijf
+
+    Scenario: wel leveren wanneer de datum aanvang van de technisch gewijzigde verblijfplaats later is dan de aanvang oorspronkelijke
+      Gegeven de persoon met burgerservicenummer '000000012' is ingeschreven op adres 'A1' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0800                              | 20020701                           |
+      En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
```

Comment:
Bij bewoning is er een stap `Gegeven adres '...' is op '...' geactualiseerd met de volgende gegevens`. Is het gedrag van die stap anders dan wat hier wordt beoogd?

Reply:
Verder vraag ik mij af of in dit geval aangifte adreshouding de waarde 'T' kan hebben. Als ik het LO goed begrijp heeft aangifte adreshouding deze waarde als adresgegevens wordt geactualiseerd. Dit is het geval als alleen de groepen Gemeente of Adres of Locatie worden gewijzigd. Zie paragraaf 3.9.7. Volgens mij is dit het geval in deze scenario. Ook lees ik op pagina 270 dat bij een actualisering van verblijfplaats datum aanvang adreshouding niet wijzigt. De datum wordt overgenomen van de verblijfplaats die door actualisering historisch wordt.

Reply:
ik zou inderdaad de stap `Gegeven adres '...' is op '...' geactualiseerd met de volgende gegevens` kunnen gebruiken. pas ik aan

Reply:
Ik vraag mij wel af of het dan klopt om te zeggen dat een persoon op twee adressen heeft verbleven, terwijl het een geactualiseerd adres is en dus in principe hetzelfde adres is. In deze scenario is dat niet duidelijk te zien, maar in de periode variant van de scenario worden twee verblijfplaats voorkomens geleverd als de gevraagde periode de datum ingang infrastructurele wijziging (= datum aanvang volgende verblijf) overlapt.